stdlib/public/SDK/Foundation/NSDictionary.swift

```diff
@@ -34,7 +34,7 @@ extension Dictionary {
   ///
   /// The provided `NSDictionary` will be copied to ensure that the copy can
   /// not be mutated by other code.
-  public init(_cocoaDictionary: __shared _NSDictionary) {
+  fileprivate init(_cocoaDictionary: __shared _NSDictionary) {
```

Comment:
This *used* to be an entry point to bridging as far as Philippe and I can remember. This is no longer used anywhere except in this file. (Same with the `NSSet` change.)